sources/utils/file_handler.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+)
+
+//FileHandler decide how to convert a file content into key values
+//archaius will manage file content as those key values
+type FileHandler func(filePath string, content []byte) (map[string]interface{}, error)
+
+//Convert2JavaProps is a FileHandler
+//it convert the yaml content into java props
+func Convert2JavaProps(p string, content []byte) (map[string]interface{}, error) {
```

Comment:
since you moves code here ,you should delete same code in file source package and change the import packge